lib/git-hooks.rb

```diff
@@ -24,9 +24,12 @@ class << self
     attr_writer :configurations
 
     def execute_pre_commits
-      configurations.pre_commits.each do |pre_commit|
+      configurations.pre_commits.each do |pre_commit, options|
         puts "Executing #{pre_commit}"
-        GitHooks::PreCommit.const_get(pre_commit).validate
+        klass = GitHooks::PreCommit.const_get(pre_commit)
+
+        next klass.validate(options) if options
```

Comment:
hhmm...

Probably we should rename{klass,pre_commit_validator}. Or something along those lines